### PR TITLE
Updating template lint errors

### PR DESCRIPTION
**Pull request by kiwiupover**

### CODE REVIEW

File: app/templates/community/meetups/index.hbs [hunk 1]
```diff
@@ -46,13 +46,19 @@
 
   <section aria-labelledby="start-your-meetup">
     <h2 id="start-your-meetup">Start your own meetup</h2>
-    <img alt="Meetup" src="/images/community/meetup.png" align="left" role="presentation">
+    <img alt="Meetup" src="/images/community/meetup.png" align="left" loading="lazy">
```
Comment (kiwiupover) [inline app/templates/community/meetups/index.hbs:49]: An image with an `alt` that has content should not have a `role="presentation"`

File: app/components/project-listing/template.hbs [hunk 2]
```diff
@@ -4,9 +4,9 @@
 
 {{#each @projects as |project|}}
   <div class="ember-cli-usage">
-    {{ember-cli-usage
-    project=project
-    channel=@channel}}
+    <EmberCliUsage
+      project={{@project}}
+      channel={{@channel}} />
```
Comment (ijlee2) [inline app/components/project-listing/template.hbs:9]: `project` came from the `{{each}}` loop.
```suggestion
      @project={{project}}
      @channel={{@channel}} />
```

File: app/templates/community/meetups/index.hbs [hunk 3]
```diff
@@ -4,37 +4,37 @@
...
-      {{#leaflet-map class="leaflet-container" lat=lat lng=lng zoom=zoom}}
-        {{tile-layer url="https://{s}.basemaps.cartocdn.com/light_all/{z}/{x}/{y}.png"}}
+    {{#if this.leafletPackageLoaded}}
+      <LeafletMap class="leaflet-container" @lat={{this.lat}} @lng={{this.lng}} @zoom={{this.zoom}}>
+        <TileLayer @url="https://{s}.basemaps.cartocdn.com/light_all/{z}/{x}/{y}.png"/>
 
-        {{#each model as |meetup|}}
-          {{#marker-layer lat=meetup.lat lng=meetup.lng}}
-            {{#popup-layer}}
+        {{#each this.model as |meetup|}}
```
Comment (ijlee2) [inline app/templates/community/meetups/index.hbs:11]: Can we use `@model` to clarify that we don't transform what the `model` hook returns in the controller?
```suggestion
        {{#each @model as |meetup|}}
```

File: app/templates/community/meetups/index.hbs [hunk 4]
```diff
@@ -4,37 +4,37 @@
...
-      {{#each sortedMeetupsByArea as |area|}}
+      {{#each this.sortedMeetupsByArea as |area|}}
         <div class="mt-3">
           <h3>{{area.value}}</h3>
           <ul class="list-unstyled grid lg:grid-4 sm:grid-2">
             {{#each area.items as |meetup|}}
               <li>
-                <img width="12" class="align-middle icon-left" src="/images/meetups/map-pin.png" alt="location icon" role="presentation">
+                <img width="12" class="align-middle icon-left" src="/images/meetups/map-pin.png" alt="location icon" loading="lazy">
                 <a href={{meetup.url}}>{{meetup.location}}</a>
```
Comment (ijlee2) [inline app/templates/community/meetups/index.hbs:38]: Unrelated to your changes, but I wondered if we need to open the link in a new tab, just like lines 14-19.

**Comment by kiwiupover** [comment]
Thanks heaps @ijlee2 Isaac I'm going to close this PR and make sure I have the app running correctly locally. 

I can't get the CSS working.

**Pull request closed without merging**